FileShareClient/Models/BatchAttributesSearchResponse.cs

```diff
@@ -0,0 +1,76 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+using System.Text;
+
+namespace UKHO.FileShareClient.Models
+{
+    [DataContract]
+    public class BatchAttributesSearchResponse : IEquatable<BatchAttributesSearchResponse>
+    {
+        public BatchAttributesSearchResponse(int searchBatchCount = default, List<Attributes> batchAttributes = default)
+        {
+            SearchBatchCount = searchBatchCount;
+            BatchAttributes = batchAttributes;
+        }
+
+        [DataMember(Name = "searchBatchCount", EmitDefaultValue = false)]
+        public int? SearchBatchCount { get; set; }
+
+        [DataMember(Name = "batchAttributes", EmitDefaultValue = false)]
+        public List<Attributes> BatchAttributes { get; set; }
+
+        public override string ToString()
```

Comment:
Please use `nameof` wherever possible in `ToString` and also add a covering test to verify `BatchAttributes` is printed as expected (i.e. attribute values printed,  not the type name)